backends/intel_gpu/CMakeLists.txt

```diff
@@ -0,0 +1,148 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(paddle-custom-intel-gpu CXX C)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
+
+option(WITH_ONEDNN "use oneDNN with custom kernels" OFF)
+option(WITH_KERNELS "compile with custom kernels" ON)
+option(WITH_TESTING "compile with unit testing" ON)
+
+if(NOT DEFINED ONEAPI_PATH)
+  set(ONEAPI_PATH "$ENV{HOME}/intel/oneapi")
+endif()
+
+if(NOT EXISTS "${ONEAPI_PATH}")
+  message(FATAL_ERROR "Can't find openapi directory ${ONEAPI_PATH}")
+endif()
+
+message(STATUS "OneAPI DIR : ${ONEAPI_PATH}")
+
+if(NOT DEFINED ONEAPI_COMPILER_DIR)
+  set(ONEAPI_COMPILER_DIR "compiler/latest")
+endif()
+
+if(NOT EXISTS "${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}")
+  message(
+    FATAL_ERROR "Can't find directory ${ONEAPI_PATH}/${ONEAPI_COMPILER_DIR}")
+endif()
+
+if(NOT DEFINED ENV{CONDA_PREFIX})
+  message(FATAL_ERROR "Please install python miniconda ")
+endif()
+
+if(NOT DEFINED CND_INSTALL_DIR)
+  set(CND_INSTALL_DIR "$ENV{CONDA_PREFIX}")
+endif()
+
+if(NOT EXISTS "${CND_INSTALL_DIR}")
+  message(FATAL_ERROR "Can't find conda directory ${CND_INSTALL_DIR}")
+endif()
+
+if(NOT DEFINED ENV{DPL_ROOT})
+  message(FATAL_ERROR "Please set the env var DPL_ROOT!" $ENV{DPL_ROOT})
+endif()
+message(STATUS "Conda DIR : ${CND_INSTALL_DIR}")
+
+set(CND_PYTHON "${CND_INSTALL_DIR}/lib/python3.9")
+
+if(NOT EXISTS "${CND_PYTHON}")
+  message(
+    FATAL_ERROR
+      "Can't find conda python directory ${CND_PYTHON}, python3.9 required!")
+endif()
```

Comment:
30 - 53 lines of code is just to check the python 3.9 is required, right?

If python 3.9 is installed by yum or apt instead of conda, it should also be suitable?

If yes, please replace 30 - 53 to reduce limitation of user environment, just to check python 3.9 required such as 


```
find_package(Python3 3.9 EXACT)
```

Reply:
Hi @qili93  seems cmake orignal find_package() not friendly for conda, could we pending this with next pr with docker image together? cc: @Zhiwei35

Reply:
done